sled-agent/src/metrics.rs

```diff
@@ -438,6 +520,14 @@ impl MetricsRequestQueue {
         }
         success
     }
+
+    /// Notify the task that a sled's synced value has changed.
+    ///
+    /// Typically, this is used to notify the task that a sled has *indeed* been
+    /// synced with NTP.
+    pub async fn synced_sled(&self, sled_id: Uuid, synced: bool) -> bool {
```

Comment:
> I'm still returning the bool, but logging at the call site if it does error 

Ok, that makes sense.

> Should we retry vs. lose the metrics completely?

This queue is backed by a `tokio::sync::mpsc::Sender`. That only fails if the receive side is closed, which means the task exited and the rest of the sled-agent is probably going to follow soon. I think retrying will result in the same error. This was put here to log the fact that this happened, rather than the expectation that we might respond to this kind of error.

Reply:
Yeah, agreed.